app/models/course.rb

```diff
@@ -15,6 +15,7 @@ class Course < ApplicationRecord
     further_education: 'Further education',
   }, _suffix: :course
 
+  scope :that_candidates_can_apply_to, -> { where(exposed_in_find: true) }
```

Comment:
There are two attributes on Find courses:

* `findable?` - these courses are listed on Find (with or without vacancies)
* `open_for_applications?` - these courses are listed on Find and can be applied to (i.e. definitely have vacancies)

so the labelling isn't quite correct

Reply:
Would it be confusing for someone to arrive from Find with a `findable?` course and then not be able to summon it using the find-a-course UI on the Apply side?

Reply:
The Apply action isn't available on Find if `open_for_applications?` is false.

Reply:
My intention was for that scope to gain additional checks, like whether or not the provider is whitelisted to do Dual running (part of https://trello.com/c/5wXHsiOC). That's why the more generic name. 

I've also [added a story to the candidate board](https://trello.com/c/LAh0iGIc/348-only-allow-candidates-to-apply-to-courses-that-have-spaces-available) to show/hide courses based on the availibility - I don't think that was previously planned in.

Does that make sense?

Reply:
Uhh still uneasy with a mismatch between naming and what it actually does – can't it just be renamed at the same time as you broaden the scope?

Reply:
It's not a mismatch, it accurately describes the situation: you can really apply to all these courses in the current version of Apply.

Removed the scope here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/506/commits/ac8f42b3643f846a7669fcb3885a559bd46b9d21

Reply:
Can apply ≠ should be able to apply, thanks for taking the scope out.